test/testgroups/invalidopsrctests.go

```diff
@@ -0,0 +1,15 @@
+package testgroups
+
+import (
+	"testing"
+
+	"github.com/operator-framework/operator-marketplace/test/testsuites"
+)
+
+// InvalidOpSrcTestGroup runs test suites for invalid operatorsources
+func InvalidOpSrcTestGroup(t *testing.T) {
```

Comment:
`Test Groups` are typically used to create any resources that testsuites / tests in the group depend on. In this case, you've created a `test group` called `InvalidOpSrcTestGroup` - I think it makes more sense to create a `test group` meant for test suites that require no set up.

Reply:
Following the conversation on Kevin's PR, I'll update my tests to follow this format.